genrl/deep/agents/a2c/a2c.py

```diff
@@ -116,6 +123,27 @@ def create_model(self) -> None:
 
         self.rollout = RolloutBuffer(self.rollout_size, self.env)
 
+    def create_custom_model(self) -> None:
+        """
+        Creates custom actor critic model and initialises optimizers
+        """
+        if self.noise is not None:
+            self.noise = self.noise(
+                np.zeros_like(action_dim), self.noise_std * np.ones_like(action_dim)
+            )
+        try:
+            self.ac = self.network_type(**kwargs).to(self.device)
+
+            self.optimizer_policy = opt.Adam(
+                self.ac.actor.parameters(), lr=self.lr_policy
+            )
+            self.optimizer_value = opt.Adam(
+                self.ac.critic.parameters(), lr=self.lr_value
+            )
+            self.rollout = RolloutBuffer(self.rollout_size, self.env)
+        except KeyError:
```

Comment:
It's not recommended to use try except within scripts, can you find some workaround for this?